pkgs/development/tools/misc/pwndbg/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitHub, pythonPackages, makeWrapper, gdb }:
+
+stdenv.mkDerivation rec {
+  name = "pwndbg-2018-04-06";
+
+  src = fetchFromGitHub {
+    owner = "pwndbg";
+    repo = "pwndbg";
+    rev = "e225ba9f647ab8f7f4871075529c0ec239f43300";
+    sha256 = "1s6m93qi3baclgqqii4fnmzjmg0c6ipkscg7xiljaj5z4bs371j4";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  propagatedBuildInputs = with pythonPackages; [
+    future
+    isort
+    psutil
+    pycparser
+    pyelftools
+    python-ptrace
+    ROPGadget
+    six
+    unicorn
+    pygments
+    enum34
+  ];
+
+  installPhase = ''
+    mkdir -p $out/share/pwndbg
+    cp -r *.py pwndbg $out/share/pwndbg
+    makeWrapper ${gdb}/bin/gdb $out/bin/pwndbg \
+      --add-flags "-q -x $out/share/pwndbg/gdbinit.py"
+  '';
+
+  preFixup = ''
+    sed -i "/import sys/a import sys; sys.path.insert(0, '$PYTHONPATH'.split(':'))" \
```

Comment:
This `insert` looks wrong, since `insert` takes a single element to insert (not a list of elements). `sys.path[0:0] = '$PYTHONPATH'.split(':')` should do what we want though

Reply:
The weird thing seems, that it seems not to make a difference.

Reply:
For me, it doesn't work with the insert. Maybe you have the packages already installed in your global python env? I'll push the fix.

Reply:
maybe, I should probably remove pwntools again.